### DISCUSSION TITLE
Fungsi menghapus data

### DISCUSSION THREAD

**Discussion by Farabisuhud** [teamalgoritma/community#207] [Q&A] [answered]
Bagaimana cara untuk menghapus data yang missing atau duplikasi? Syntax apa yang harus digunakan dan bagaimana cara menggunakan data yang sudah di clean untuk digunakan selanjutnya

**Comment by inytss** [comment, marked as answer]
Dear @Farabisuhud ,
Untuk menghapus data duplicate bisa menggunakan conditional subsetting yaitu `data[!duplicated(data),]` sedangkan untuk menghapus missing value berdasarkan baris bisa menggunakan `na.omit()`. Atau selengkapnya dapat Bapak dapat mengeskplore handling missing value pada laman berikut https://askalgo.netlify.app/#handling-missing-value

Apabila jawaban berikut membantu, silahkan boleh menekan tombol "Mark as answered". Terima kasih.
